hello_server.go

```diff
@@ -30,18 +31,19 @@ func CreateGreeting(name string) string {
 	return "Hello, " + name + "\n"
 }
 
+func CreateGreeting1(name string) string {
+	if name == "" {
+		name = "Guest"
```

Comment:
1

```suggestion
      case ToolbarAction.SUGGESTION: {
        const main = view.state.selection.main
        const txt = '// your suggestion here'
```

2 

```suggestion
func CreateGreeting(name string) string {
export const CodeSuggestionBlock: React.FC<{ code: string }> = ({ code }) => {
  const html = toHtml(refractor.highlight(code, 'js') as unknown as Nodes)
```

Github does not know how to match

```suggestion
func CreateGreeting(name string) string {
export const CodeSuggestionBlock: React.FC<{ code: string }> = ({ code }) => {
  const html = toHtml(refractor.highlight(code, 'js') as unknown as Nodes)
  
```

3

```suggestion
func CreateGreeting(name string) string {
	if name == "" {
		name = "Guest"
```

Reply:
4

```suggestion
func CreateGreeting(name string) string {
	if name == "" {
		name = "Guest"
```